pkg/server/internal/extensions/extensions_test.go

```diff
@@ -65,7 +65,7 @@ var (
 )
 
 func TestValidateCSRExtentions(t *testing.T) {
-	sk, err := pkiutil.GenerateRSAPrivateKey(2048)
+	sk, err := pkiutil.GenerateRSAPrivateKey(4096)
```

Comment:
It probably doesn't make sense for us to use `4096` key sizes in tests, since it is just going to slow things down for no real benefit.